discord/api/websocket.py

```diff
@@ -41,6 +46,11 @@ def __init__(self, client, token: str) -> None:
         self.heartbeat_acked = True
         self.closed: bool = False
 
+        self.guild_cache = LFUCache[Snowflake, Guild](1000)
```

Comment:
We should decide on the cache size tbh

Reply:
maybe make it so that the user can input the cache size?